Users/Install.rst

```diff
@@ -70,6 +70,8 @@ To install coala only (without any bears), you can do:
 
     $ pip3 install coala
 
+If, after installing coala, you install Java (information for downloading Java for all majors OS can be found `here <https://www.java.com/en/download/help/download_options.xml>`_.), you must restart your shell. This is to inform coala that Java has been installed.        
```

Comment:
so I'd make this a note and make the phrasing a bit less complicated, shorter and include reasoning for needing java because a lot of people don't like it, how about something along the lines of:

> Be sure to have java installed if you install the coala-bears package. You might need to restart your shell after installing it so coala picks it up. Java is needed for our grammar checking analysis routines.